huxley/core/models.py

```diff
@@ -394,6 +394,11 @@ class Delegate(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     summary = models.TextField(default='', blank=True, null=True)
 
+    friday_attendance = models.BooleanField(default=False)
```

Comment:
a more manageable naming scheme might be "session_one, session_two, session_three, and session_four". They are technically committee sessions.